core-services/image-mirroring/_config.yaml

```diff
@@ -523,8 +523,8 @@ supplementalCIImages:
     image: quay.io/stackrox-io/ci:automation-flavors-aro-0.9.1
   stackrox/automation-flavors:rosa-0.9.1:
     image: quay.io/stackrox-io/ci:automation-flavors-rosa-0.9.1
-  stackrox/automation-flavors:ibmcloudz-0.9.1:
-    image: quay.io/stackrox-io/ci:automation-flavors-ibmcloudz-0.9.1
```

Comment:
You have to keep the old ones for existing dependencies not to break during the switch.

Reply:
Understood. I kept the old one.